lumicks/pylake/kymo.py

```diff
@@ -299,11 +330,20 @@ def _plot(self, channel, axes, adjustment=ColorAdjustment.nothing(), **kwargs):
             cmap=linear_colormaps[channel],
         )
 
-        image_handle = axes.imshow(image, **{**default_kwargs, **kwargs})
+        image_handle = show_image(
+            image,
+            adjustment,
+            channel,
+            image_handle=image_handle,
+            axes=axes,
+            **{**default_kwargs, **kwargs},
+        )
         axes.set_xlabel("time (s)")
         axes.set_ylabel(f"position ({self._calibration.unit_label})")
-        axes.set_title(self.name)
-        adjustment._update_limits(image_handle, image, channel)
+        if show_title:
+            axes.set_title(self.name)
+
+        return image_handle
```

Comment:
so technically adding this I guess could be considered a breaking change 😕 but this would only affect things if someone was checking that the returned value was `None` which would've always be `True` which is .... less than useful. I say we keep this without doing a breaking release, because it's nice to have consistency, but let's put a note in the changelog to just highlight the change

Reply:
I'd be fine with it.

Reply:
Added a note in the changelog.